packages/sdk/src/oracle/README.md

```diff
@@ -12,14 +12,25 @@ import {oracle} from '@uma/sdk'
 // create your configuration object with known addresses
 type Config = oracle.types.state.Config
 
-// creating a config for mainnet usage add any additional chains you want
+// creating a config for mainnet usage add any additional chains you want following the same format
 export const config = {
   chains: {
     1:{
       chainId: 1,
-      multicall2Address: process.env.multicall2Address,
-      optimisticOracleAddress: process.env.optimisticOracleAddress,
+      multicall2Address: "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
+      optimisticOracleAddress: "0xC43767F4592DF265B4a9F1a398B97fF24F38C6A6",
       providerUrl: process.env.CUSTOM_NODE_URL,
+      metadata:{
+        chainId: 1,
+        rpcUrls: [],
+        nativeCurrency: {
+          name: "Ether",
+          symbol: "ETH",
+          decimals: 18,
+        },
+        chainName: "Ethereum",
+        blockExplorerUrls: "https://etherscan.io"
+      }
```

Comment:
this was mainly tacked on as the exact data needed for requesting a user add a new provider through metamask. I agree theres a bunch of redundant fields, it might make sense to make this the default object instead.

Reply:
this data structure is based on this https://eips.ethereum.org/EIPS/eip-3085 so its not strictly a frontend thing